specs/src/specs/block-manager.md

```diff
@@ -1 +0,0 @@
-../../../block/block-manager.md
```

Comment:
We should just ignore these symlink files in the CI instead of doing this. We wanted to use symlinks in the specs directory so that the specs directory is just a shell for the specs site to use, and then the specs live with the code which is easier for maintenance long term.